docs/howto/windows.md

```diff
@@ -106,28 +87,132 @@ many of these.)
-the same steps as in [our main setup guide](build-run.md) -- just use the
-Linux versions. :-)
+### Step-by-step instructions for setting up WSL
+
+1. Open Powershell as an Administrator and run the command below to enable WSL. 
+
+    `Enable-WindowsOptionalFeature -Online -FeatureName Microsoft-Windows-Subsystem-Linux`
+
+2. Install [Chocolatey][chocolatey]. After installing Chocolatey, use it to install the JAVA SE Development Kit with  
+
+    `choco install -y jdk8`
+
+3. Follow the [instructions][android-studio] here to install Android Studio. 
+
+4. In Step 3 above, you saw how to add the `ANDROID_HOME` environment variable. Return to the Environment Variables screen. Scroll down in the User variables section where you added `ANDROID_HOME` until you find a variable called `Path`. 
+
+    * Double-click on it, click New in the window that comes up and add the line below: 
+
+        `%ANDROID_HOME%\tools;%ANDROID_HOME%\platform-tools;%ANDROID_HOME%\build-tools;`
+
+    * Then move to the System variables section and add the `ANDROID_HOME` environment variable you added at the end of Step 3 to System variables. 
 
-Similarly, when the React Native install instructions say to install the
-`react-native` CLI with `npm install -g react-native-cli`, run that command
-in the Linux environment.
+    * Finally, scroll down in the System variables section until you find `Path`. Double-click on it, click New in the window that comes up and then add the lines below in separate lines. Be sure to replace `{User}` in the lines below with your Windows account username. 
 
-Now `yarn` and `react-native` should be available at your `bash` prompt.
+        ```
+        C:\Users\{User}\AppData\Local\Android\Sdk\platform-tools
+        C:\Users\{User}\AppData\Local\Android\Sdk\emulator
+        ```
 
-### Modify PATH variable
-Run the following commands so Linux programs check for `yarn`, etc., in
+
+5. Go to the Windows Store and type the name of the linux distro you would like to use with WSL. These instructions have been tested to work on Ubuntu and DebianGNU/Linux. When the page for your selected distro opens. Click on 'Get' to install it. 
+
+6. You will receive a notification when the installation has completed. Click on 'Launch' to open it. When it opens, you will see the message "Installing, this may take a few minutes...". Soon after that, you will be prompted to enter the username and password you wish to use. If you haven't used a UNIX based system you will notice that nothing shows up while you are typing your password. This is normal and expected behavior.  
+
+7. After setting up your username and password, run 
+
+    `sudo apt update && sudo apt -y upgrade`
+
+    This will initialize your WSL installation and update your distro. This could take a while so grab a cup of coffee. (You can paste this and the rest of the commands in this guide into the WSL window using your right-click button. Ctrl + V will not work. )
```

Comment:
Huh interesting. Important tip!

Does Shift + Insert work? (That's an alternate key combo for "paste" that I remember using a lot on Windows. Also Ctrl + Insert for "copy" and Shift + Delete for "cut", if I recall right.) If so, that might be worth mentioning too.

Reply:
I tried those out but they don't work. I discovered the right click trick by accident so I never went on to search for a solution for that. Just did and found that there's an [option](https://blogs.msdn.microsoft.com/commandline/2018/04/13/copy-and-paste-arrives-for-linuxwsl-consoles/) to enable "Ctrl+Shift+C/V" for copy paste so I'll include a link to that in the next version of the doc.